collectors/proc.plugin/proc_loadavg.c

```diff
@@ -113,10 +127,13 @@ int do_proc_loadavg(int update_every, usec_t dt) {
             );
 
             rd_active = rrddim_add(processes_chart, "active", NULL, 1, 1, RRD_ALGORITHM_ABSOLUTE);
+            rd_pidmax = rrddim_add(processes_chart, "pidmax", NULL, 1, 1, RRD_ALGORITHM_ABSOLUTE);
         }
         else rrdset_next(processes_chart);
 
         rrddim_set_by_pointer(processes_chart, rd_active, active_processes);
+        rrddim_set_by_pointer(processes_chart, rd_pidmax,    max_processes);
+        rrddim_hide(processes_chart, "pidmax");
```

Comment:
could be chart [VARIABLE](https://learn.netdata.cloud/docs/agent/collectors/plugins.d#variable)

i think it is better then a hidden dimension

Reply:
But then should we implement another notification? Because the `$pidmax` will not be available to calculate on `processes.conf` otherwise

Reply:
> VARIABLE defines a variable that can be used in alarms

If we dont need history but the current value only VARIABLE is the way to go

Reply:
Okay, I will look into it and add it! (should I enter WIP in the title as I'm changing the code again?)

Reply:
I made it `draft`, click on `Ready for review` button when the PR is done 👍

Reply:
Perfect, thanks!